src/main/java/com/example/PhysioCenter/service/serviceImpl/VisitServiceImpl.java

```diff
@@ -0,0 +1,85 @@
+package com.example.PhysioCenter.service.serviceImpl;
+
+import com.example.PhysioCenter.domain.dto.patient.PatientDto;
+import com.example.PhysioCenter.domain.dto.physioteraphist.PhysioDto;
+import com.example.PhysioCenter.domain.dto.visit.DateOfTheVisitDto;
+import com.example.PhysioCenter.domain.dto.visit.VisitDto;
+import com.example.PhysioCenter.domain.dto.visit.VisitRoomDto;
+import com.example.PhysioCenter.domain.entity.DateOfTheVisit;
+import com.example.PhysioCenter.domain.entity.Visit;
+import com.example.PhysioCenter.domain.repository.*;
+import com.example.PhysioCenter.service.VisitService;
+import com.vladmihalcea.hibernate.type.range.Range;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDateTime;
+import java.util.Date;
+import java.util.List;
+
+@Service
+public class VisitServiceImpl implements VisitService {
+
+    private final VisitRepository visitRepository;
+
+    private final PatientServiceImpl patientService;
+    private final PhysioServiceImpl physioService;
+    private final DateOfTheVisitImpl dateOfTheVisit;
+    private final RoomServiceImpl roomService;
+    private final DateOfTheVisitRepository dateOfTheVisitRepository;
+
+    public VisitServiceImpl(VisitRepository visitRepository, PatientServiceImpl patientService, PhysioServiceImpl physioService, DateOfTheVisitImpl dateOfTheVisit, RoomServiceImpl roomService, DateOfTheVisitRepository dateOfTheVisitRepository) {
+        this.visitRepository = visitRepository;
+        this.patientService = patientService;
+        this.physioService = physioService;
+        this.dateOfTheVisit = dateOfTheVisit;
+        this.roomService = roomService;
+        this.dateOfTheVisitRepository = dateOfTheVisitRepository;
+    }
+
+    @Override
+    public List<VisitDto> findAll() {
+
```

Comment:
skoro jest visitRepository, to można zaimplementować również tą metodę

Reply:
zrobi się w swoim czasie :p